packages/core/src/common/commands-common.ts

```diff
@@ -9,6 +9,16 @@ import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
 
+const enjson = require('./i18n/en.json');
+const esjson = require('./i18n/es.json');
+const frjson = require('./i18n/fr.json');
+const Globalize = require("globalize");
+
+// Load potential languages
+Globalize.loadMessages(enjson);
+Globalize.loadMessages(esjson);
+Globalize.loadMessages(frjson);
```

Comment:
It would be manageable somehow, but if we were using the transifex service with an automated script to pull the translations for a package, that process coudl also generate a `.js` module that hides the loading and initialization and exposes the two or three methods we need to localize strings.